Dockerfile

```diff
@@ -13,19 +13,17 @@ RUN echo 'APT::Install-Recommends 0;' >> /etc/apt/apt.conf.d/01norecommends && \
     echo 'APT::Install-Suggests 0;' >> /etc/apt/apt.conf.d/01norecommends && \
     apt-get -q update && \
     apt-get -q install locales && \
-    locale-gen en_US.UTF-8
-
-ENV LANG="en_US.UTF-8" \
+    locale-gen en_US.UTF-8 && \
+    LANG="en_US.UTF-8" \
```

Comment:
Those variables are not exported or anything and do nothing from that point on.
They must be defined by `ENV` or at least `export`ed though `ENV` is the better option here to have it "globally".